### DISCUSSION TITLE
Exodus Output Specific ID

### DISCUSSION THREAD

**Discussion by maxnezdyur** [idaholab/moose#21101] [Q&A General] [answered]
Say I have a model with multiple subdomains, but I only want to output a specific subdomain/block id. Is there a way to do that in MOOSE? I looked through the documentation but it doesn't look like the exists in the docs.

**Comment by GiudGiud** [comment, marked as answer]
The objects that will allow you (for which it makes sense) to do that will have a `block = ` parameter
